compiler/include/view.h

```diff
@@ -36,11 +36,8 @@ Expr*    aidExpr(int id);
 BaseAST* aid(BaseAST* ast);
 Expr*    aidExpr(BaseAST* ast);
 
-void        list_view_noline(const BaseAST* ast);
-void        nprint_view(BaseAST* ast);
-
-// defined in codegen/codegen.cpp
-void        nprint_view(GenRet& gen);
+bool inTree(BaseAST* ast);
+bool inTree(int id);
```

Comment:
Ideally these changes would be in a different PR as they aren't specific to the CG effort, right?